contracts/paratii/VideoStore.sol

```diff
@@ -19,6 +19,9 @@ contract VideoStore is Ownable, Debug {
 
     ParatiiRegistry public paratiiRegistry;
 
+    mapping (address => bytes32[]) public user_purchases;
+    mapping (bytes32 => address[]) public video_sales;
```

Comment:
Ok, so duplicating data does resolve the problem of "symmety" of the data, but it is expensive to duplicate data, and it also is a potential source of bugs (accidentally update user_purches without updating video_sales, and you have inconsistent data). 

If we only need to _check_ wheter a user purches a video or not, we can probably do with one single data structure. (i.e. drop `video_sales`). The question is then whether there are use cases where we need to do more than just checking if the purchase has occurred or not.

Reply:
`UserRegistry` stores videos liked or disliked by the address of the user and currently has an unused `isAcquired` variable in the video struct. I'm getting rid of the `user_purchases` and integrating the `isAcquired` variable into the registration.